docs/src/api.md

```diff
@@ -37,9 +52,21 @@ genocide!
 sample!
 ```
 
+### `@agents` macro
```

Comment:
Should this section be moved to above "Adding agents"? Since this is ultimately "creating agents".

Reply:
good point